### ISSUE TITLE
TypeError: Cannot read property 'index' of undefined

### ISSUE THREAD

**Issue by linfaxin** [isomorphic-git/isomorphic-git#1124]
TypeError: Cannot read property 'index' of undefined
    at Function.acquire (src/managers/GitIndexManager.js:49)
    at _commit (src/commands/commit.js:62)
    at _merge (src/commands/merge.js:140)
    at async _pull (src/commands/pull.js:96)
    at async Module.pull

The "cache" param may miss when call `_merge()` in `_pull(src/commands/pull.js:96)`.

**Comment by linonetwo** [comment]
This is also causing

```
[1] (node:31230) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'index' of undefined
[1]     at Function.acquire (/Users/linonetwo/Desktop/repo/TiddlyGit-Desktop/node_modules/isomorphic-git/index.cjs:842:16)
[1]     at _commit (/Users/linonetwo/Desktop/repo/TiddlyGit-Desktop/node_modules/isomorphic-git/index.cjs:4338:26)
[1]     at _merge (/Users/linonetwo/Desktop/repo/TiddlyGit-Desktop/node_modules/isomorphic-git/index.cjs:8445:23)
[1]     at async _pull (/Users/linonetwo/Desktop/repo/TiddlyGit-Desktop/node_modules/isomorphic-git/index.cjs:8556:5)
```

**Comment by wmhilton** [comment]
Ah, #1189 mispelled "fixes" as "fixs" so the release bot didn't post it here but I'll post it:

:tada: This issue should be fixed in version 1.7.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/isomorphic-git/v/1.7.1)
- [GitHub release](https://github.com/isomorphic-git/isomorphic-git/releases/tag/v1.7.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by linonetwo** [comment]
OMG I've used isomorphic-git before... For some reason I forget I switched to dugite for a year, but due to https://github.com/desktop/dugite/issues/440 I decided to switch back here...
